<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaperDlgClass</class>
 <widget class="QDialog" name="PaperDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaperDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leTitle"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Authors">
     <property name="text">
      <string>Authors</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="leAuthors"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Journal</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="leJournal"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Abstract</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPlainTextEdit" name="teAbstract"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Note</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="teNote"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PaperDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>-21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PaperDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>-26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
